benchmarks/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "package"
+version = "0.1.0"
+description = "OpenSearch samples."
```

Comment:
Should this be "OpenSearch benchmarks."?

Reply:
Fixed.